modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -665,6 +667,10 @@ export default {
         itemName: `${this.moduleName}:editor`
       });
     },
+    onUpdateDocFields(value) {
```

Comment:
check if `value.data._id` is equal to `this.docFields.data._id` to prevent multiple editor modal

Reply:
I removed the need for the event bus here, see my comment in code.